packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -113,7 +113,14 @@ export default declare((api, options, dirname) => {
     );
   }
 
+  function isNamespaced(path) {
+    const binding = path.scope.getBinding(path.node.name);
+    if (!binding) return false;
+    return binding.path.isImportNamespaceSpecifier();
+  }
+
   function maybeNeedsPolyfill(path, methods, name) {
+    if (isNamespaced(path.get("object"))) return false;
```

Comment:
The `maybeNeedsPolyfill` is called in `CallExpression` and `MemberExpression` visitors. If we would like to know whether `path.get("object")` is an import namespace specifier, it suffices to test this only in `MemberExpression`

https://github.com/babel/babel/blob/8a775a32ba3dc74c3bb784550ff9a78bdc98e47d/packages/babel-plugin-transform-runtime/src/index.js#L399-L404

Reply:
Just made some experiments, `CallExpression` is called before `MemberExpression`, seems there is no way to check `isNamespaced` in `MemberExpression` and prevent transformation in `CallExpression`.